managed/models/database_test.go

```diff
@@ -316,81 +311,11 @@ func TestDatabaseChecks(t *testing.T) {
 				defer rollback()
 
 				_, err = tx.Exec(
-					"INSERT INTO agents (agent_id, agent_type, runs_on_node_id, pmm_agent_id, node_id, service_id, disabled, status, created_at, updated_at, tls, tls_skip_verify, max_query_length, query_examples_disabled, comments_parsing_disabled, max_query_log_size, table_count_tablestats_group_limit, rds_basic_metrics_disabled, rds_enhanced_metrics_disabled, push_metrics, expose_exporter) "+
-						"VALUES ('8', 'mysqld_exporter', NULL, '1', '1', '1', false, '', $1, $2, false, false, 0, false, true, 0, 0, false, false, false, false)",
+					`INSERT INTO agents (agent_id, agent_type, runs_on_node_id, pmm_agent_id, node_id, service_id, disabled, status, created_at, updated_at, tls, tls_skip_verify, qan_options, mysql_options, aws_options, exporter_options) `+
+						`VALUES ('8', 'mysqld_exporter', NULL, '1', '1', '1', false, '', $1, $2, false, false, '{"max_query_length": 0, "query_examples_disabled": false, "comments_parsing_disabled": true, "max_query_log_size": 0}', '{"table_count_tablestats_group_limit": 0}', '{"rds_basic_metrics_disabled": false, "rds_enhanced_metrics_disabled": false}', '{"push_metrics": false, "expose_exporter": false}')`,
 					now, now)
 				assertCheckViolation(t, err, "agents", "node_id_or_service_id_for_non_pmm_agent")
 			})
 		})
 	})
 }
-
-func TestDatabaseMigrations(t *testing.T) {
```

Comment:
Removed because to have this one working we need two encryption structures:
`// DefaultAgentEncryptionColumnsV3 since 3.0.0 contains all tables and it's columns to be encrypted in PMM Server DB.
var DefaultAgentEncryptionColumnsV3 = []encryption.Table{
	{
		Name:        "agents",
		Identifiers: []string{"agent_id"},
		Columns: []encryption.Column{
			{Name: "username"},
			{Name: "password"},
			{Name: "agent_password"},
			{Name: "aws_options", CustomHandler: EncryptAWSOptionsHandler},
			{Name: "azure_options", CustomHandler: EncryptAzureOptionsHandler},
			{Name: "mongo_options", CustomHandler: EncryptMongoDBOptionsHandler},
			{Name: "mysql_options", CustomHandler: EncryptMySQLOptionsHandler},
			{Name: "postgresql_options", CustomHandler: EncryptPostgreSQLOptionsHandler},
		},
	},
}

// DefaultAgentEncryptionColumns contains all tables and it's columns to be encrypted in PMM Server DB.
var DefaultAgentEncryptionColumns = []encryption.Table{
	{
		Name:        "agents",
		Identifiers: []string{"agent_id"},
		Columns: []encryption.Column{
			{Name: "username"},
			{Name: "password"},
			{Name: "agent_password"},
			{Name: "aws_access_key"},
			{Name: "aws_secret_key"},
			{Name: "azure_options", CustomHandler: EncryptAzureOptionsHandler},
			{Name: "mongo_db_tls_options", CustomHandler: EncryptMongoDBOptionsHandler},
			{Name: "mysql_options", CustomHandler: EncryptMySQLOptionsHandler},
			{Name: "postgresql_options", CustomHandler: EncryptPostgreSQLOptionsHandler},
		},
	},
}`

So test is removed and we have only one encryption struct for V3.